api/config/routes.rb

```diff
@@ -161,6 +161,7 @@
         get '/order_status/:number', to: 'order_status#show', as: :order_status
         resources :products, only: %i[index show]
         resources :taxons,   only: %i[index show], id: /.+/
+        get '/stores/:code', to: 'stores#show', as: :store
```

Comment:
@coolprobn I think is somehow conflicting with API v1 routes - hence the failing tests, let's leave it as 

```suggestion
        resources :stores, only: :show
```

Reply:
Test seems to be failing after we added ability of the user to view store without login so they can access particular store data by it's code.
In V1 API, user needs to be authorized to view the store details, but for the storefront we need the data of store to be accessible without logging in.
Maybe we should update the tests, what would you suggest?

Reply:
Hello @damianlegawiec
Any suggestions?